test/support/generator/create-facility-fixed-fees-amount-amendment.generator.ts

```diff
@@ -0,0 +1,106 @@
+import { ENUMS, PROPERTIES } from '@ukef/constants';
+import { AcbsPartyId, DateOnlyString, UkefId } from '@ukef/helpers';
+import { AcbsCreateBundleInformationRequestDto } from '@ukef/modules/acbs/dto/acbs-create-bundle-information-request.dto';
+import { FacilityFeeAmountTransaction } from '@ukef/modules/acbs/dto/bundle-actions/facility-fee-amount-transaction.bundle-action';
+import { DateStringTransformations } from '@ukef/modules/date/date-string.transformations';
+import {
+  CreateFixedFeeAmountAmendmentRequest,
+  CreateFixedFeeAmountAmendmentRequestItem,
+} from '@ukef/modules/facility-fixed-fee/dto/create-facility-fixed-fee-amount-amendment-request.dto';
+
+import { AbstractGenerator } from './abstract-generator';
+import { RandomValueGenerator } from './random-value-generator';
+
+export class CreateFacilityFixedFeesAmountAmendmentGenerator extends AbstractGenerator<GenerateValues, GenerateResult, GenerateOptions> {
+  constructor(protected readonly valueGenerator: RandomValueGenerator, protected readonly dateStringTransformations: DateStringTransformations) {
+    super(valueGenerator);
+  }
+
+  protected generateValues(): GenerateValues {
+    return {
+      partyIdentifier: this.valueGenerator.acbsPartyId(),
+      period: this.valueGenerator.string({ length: 2 }),
+      lenderTypeCode: this.valueGenerator.enumValue(ENUMS.LENDER_TYPE_CODES),
+      effectiveDate: this.valueGenerator.dateOnlyString(),
+      amountAmendment: this.valueGenerator.float({ min: 0.01 }),
+    };
+  }
+
+  protected transformRawValuesToGeneratedValues(values: GenerateValues[], { facilityIdentifier }: GenerateOptions): GenerateResult {
+    const increaseAmountRequest: CreateFixedFeeAmountAmendmentRequestItem[] = values.map((value) => ({
```

Comment:
Could we change `value` to something more descriptive, e.g. `fixedFeeAmountAmendment`?